binary/src/Cardano/Binary/FromCBOR.hs

```diff
@@ -577,9 +577,9 @@ instance FromCBOR UTCTime where
       (picosecondsToDiffTime timeOfDayPico)
 
 -- | Convert an 'Either'-encoded failure to a 'cborg' decoder failure
```

Comment:
`Convert an 'Either'-encoded failure to a MonadFail`  ?

Reply:
Good catch. Fixed